core/src/main/java/nu/yona/server/subscriptions/entities/BuddyDeviceChangeMessage.java

```diff
@@ -0,0 +1,109 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Stichting Yona Foundation This Source Code Form is subject to the terms of the Mozilla Public License, v.
+ * 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *******************************************************************************/
+package nu.yona.server.subscriptions.entities;
+
+import java.util.Optional;
+import java.util.UUID;
+
+import javax.persistence.Entity;
+import javax.persistence.Transient;
+
+import nu.yona.server.crypto.seckey.SecretKeyUtil;
+import nu.yona.server.device.service.DeviceChange;
+import nu.yona.server.messaging.entities.BuddyMessage;
+
+@Entity
+public class BuddyDeviceChangeMessage extends BuddyMessage
+{
+	@Transient
+	private DeviceChange change;
+	private byte[] changeCiphertext;
```

Comment:
Do we really have to encrypt the change? Seems superfluous

Reply:
I agree. Changed it.